CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ project(
   LANGUAGES C CXX
   DESCRIPTION "MQT SyReC Synthesizer: A Tool for HDL-based Synthesis of Reversible Circuits")
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
I think I'd rather have these properties set on a per-target basis similar to how we do this in MQT Core.
See https://github.com/munich-quantum-toolkit/core/blob/5d857ecf70bbd28f295d39ded773d9aacb03e05c/cmake/CompilerOptions.cmake#L66-L68 for IPO and https://github.com/munich-quantum-toolkit/core/blob/5d857ecf70bbd28f295d39ded773d9aacb03e05c/src/ir/CMakeLists.txt#L29-L30 for the C++ standard

Reply:
Nice 🎉